config/locales/en_GB.yml

```diff
@@ -1036,6 +1092,19 @@ en_GB:
   ticket_column_item: "Item"
   ticket_column_unit_price: "Unit Price"
   ticket_column_total_price: "Total Price"
+  menu_1_title: "Shops"
+  menu_1_url: "/shops"
+  menu_2_title: "Map"
+  menu_3_title: "Producers"
+  menu_3_url: "/producers"
+  menu_4_title: "Groups"
+  menu_4_url: "/groups"
+  menu_5_title: "About"
+  menu_5_url: "http://www.openfoodnetwork.org/"
+  menu_6_title: "Connect"
```

Comment:
this en_GB file is for https://openfoodnetwork.org.uk/ right?
to have the current uk menu we need to change these to:
menu_5_url: "http://about.openfoodnetwork.org.uk/"
  menu_6_title: "Blog"
menu_6_url: "https://about.openfoodnetwork.org.uk/blog/"
  menu_7_title: "Support"
  menu_7_url: "https://about.openfoodnetwork.org.uk/support/"